application/Modules/Clipboard/Native/ClipboardWindowMessageSink.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using MORR.Shared.Hook;
+
+namespace MORR.Modules.Clipboard.Native
+{
+    public class ClipboardWindowMessageSink
+    {
+        private static INativeClipboard nativeClipboard = new NativeClipboard();
+
+        /// <summary>
+        ///     Handles a window message when clipboard is updated
+        /// </summary>
+        /// <param name="IntPtr">The pointer to the current window</param>
+        /// <param name="messageId">The identifier of the message</param>
+        /// <param name="wParam">The WPARAM of the message</param>
+        /// <param name="lParam">The LPARAM of the message</param>
+        public delegate void ClipboardEventHandler(IntPtr hwnd, uint uMsg, IntPtr wParam, IntPtr lParam);
+
+        public INativeClipboard NativeClipboard
+        {
+            get { return nativeClipboard; }
+            private set { }
+        }
```

Comment:
This and line 8 could be replaced with `public static INativeClipboard NativeClipboard { get; } = new NativeClipboard();`